requirements.txt

```diff
@@ -25,18 +24,17 @@ cryptography==1.8.1       # via paramiko, pyopenssl
 debtcollector==1.12.0     # via oslo.config, oslo.utils, python-keystoneclient, python-neutronclient
 defusedxml==0.5.0         # via djangorestframework-xml
 deprecation==1.0          # via openstacksdk
-django-celery==3.1.16
 django-cors-headers==0.12.0
-django-cyverse-auth==1.0.10
-django-filter==0.13.0
+django-cyverse-auth==1.0.11
+django-filter==1.0.1
 django-redis-cache==0.13.0
 django-sslserver==0.19
-Django==1.9.8             # via django-cors-headers, django-sslserver
+django==1.10.6
```

Comment:
Need to upgrade dev_requirements.txt as well. See https://github.com/steve-gregory/atmosphere/pull/5